### PR TITLE
Added bullet dependency to CentOS

### PR DESCRIPTION
**Pull request by ahcorde**
Added bullet dependency to CentOS

ci job is failling https://ci.ros2.org/view/nightly/job/nightly_linux-centos_debug/

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

**Comment by cottsay** [comment]
Are you sure we don't need the headers provided by `bullet-devel`?

**Comment by ahcorde** [comment]
no sure about how yum works. I added bullet and bullet-dev

**Comment by cottsay** [comment]
There is no `bullet-dev` - the package is called `bullet-devel` on RHEL. The `-devel` package will install the non-devel package as well, so you don't need to list them both.

Looks like the dependency came from here: https://github.com/ros2/geometry2/pull/205/files#diff-18a6569bed12ff565766873c287142aeR20

There is no rosdep entry for RHEL - I'll add it to my existing RHEL rule PR. The other rules are installing headers, so `bullet-devel` is the right thing to install here.

https://github.com/ros/rosdistro/blob/ca9dce084108079a367d8ba8714815ac81fb4219/rosdep/base.yaml#L307

File: linux_docker_resources/Dockerfile-CentOS [hunk 1]
```diff
@@ -15,6 +15,7 @@ RUN yum -y copr enable cottsay/devtoolset-8-make-nonblocking
 RUN yum install \
   @buildsys-build \
   asio-devel \
+  bullet-dev \
```
Comment (cottsay) [inline linux_docker_resources/Dockerfile-CentOS]: This is still not right.
```suggestion
  bullet-devel \
```

**Comment by cottsay** [comment]
Here's the new rosdep rule for the `bullet` key: https://github.com/ros/rosdistro/pull/23957/files#diff-acba1c75aeaa27ae4af7a5e0a5b78c08R312-R313

**Pull request merged**